src/Sylius/Bundle/AdminBundle/Menu/MainMenuBuilder.php

```diff
@@ -40,8 +40,26 @@ public function createMenu()
      */
     private function addConfigurationMenu(ItemInterface $menu)
     {
-        $this->configureConfigurationSubMenu($menu);
+        $this->configureCatalogSubMenu($menu);
         $this->configureMarketingSubMenu($menu);
+        $this->configureConfigurationSubMenu($menu);
+    }
+
+    /**
+     * @param ItemInterface $menu
+     */
+    private function configureCatalogSubMenu(ItemInterface $menu)
+    {
+        $catalogSubMenu = $menu
+            ->addChild('catalog')
+            ->setLabel('sylius.menu.admin.main.catalog.header')
+        ;
+
+        $catalogSubMenu
+            ->addChild('product_options', ['route' => 'sylius_admin_product_option_index'])
+            ->setLabel('sylius.menu.admin.main.catalog.manage_product_options')
```

Comment:
In menu should we have 'Product options' or 'Manage options' like in old administrator panel?